mytools.py

```diff
@@ -23,13 +27,26 @@
 _var_CameraID = "CAMERA_ID"
 
 
+def _check_pysdk_ver():
+    """Check that PySDK version is compatible with this script"""
+    import degirum as dg  # import DeGirum PySDK
+
+    if pkg_version.parse(dg.__version__) < min_compatible_pysdk_ver:
+        raise Exception(
+            f"Currently installed Degirum PySDK version {dg.__version__} is not compatible with this script. "
+            + f"Please install PySDK version {min_compatible_pysdk_ver} or higher."
+        )
+
+
 def _reload_env(custom_file="env.ini"):
     """Reload environment variables from file
     custom_file - name of the custom env file to try first; if it is None or does not exist, `.env` file is loaded
     """
     from pathlib import Path
     import dotenv
 
+    _check_pysdk_ver()
```

Comment:
could this be part of the module (executed on import) vs in _reload_env? currently it won't be exercised by the tests as they monkeypatch _reload_env

Reply:
Done